### PR TITLE
Bambenek Consulting Feed: added the tags parameter

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [ ] Ready
- [x] In Hold - Can be merged only after this merged https://github.com/demisto/content/pull/7810

## Related Issues
fixes: https://github.com/demisto/etc/issues/22895

### CODE REVIEW

**Comment by jochman** [comment]
Fails on out-of-quota. (403)

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml [hunk 1]
```diff
@@ -6,11 +6,9 @@ configuration:
 - display: Fetch indicators
   name: feed
   required: false
-  defaultvalue: true
```
Comment (Itay4) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml:9]: why did you change all default values?

Comment (jochman) [reply Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml:9]: dsdk-format command did it. revert this?
